backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/rating/BpkRating.kt

```diff
@@ -36,9 +36,9 @@ enum class BpkRatingScale {
 
 @Composable
 fun BpkRating(
-    title: String,
     value: Float,
     modifier: Modifier = Modifier,
+    title: String? = null,
```

Comment:
This is the breaking change

Reply:
making something nullable isn't a breaking change, though - only the reverse